Sources/TrustlySDK/TrustlyView.swift

```diff
@@ -496,12 +496,13 @@ public class TrustlyView : UIView, TrustlyProtocol, WKNavigationDelegate, WKScri
             fn = "selectBank"
         }
 
-        if ("local" == env), let urlLocal = localUrl {
-            url = "http://"+urlLocal+"/start/selectBank/"+fn+"?v="+build+"-ios-sdk"
-            
-            let cleanLocalUrl = urlLocal.components(separatedBy: ":")[0]
+        if (env == "local") {
+            let domain = envHost ?? "localhost"
+            url = "http://"+domain+":8000/start/selectBank/"+fn+"?v="+build+"-ios-sdk"
```

Comment:
I think we can reduce this code if the user already provide the correct protocol for us, in this case we only need to concatenate the localUrl provided with the /start/...

Reply:
The idea here is to decrease the workload for testers. They'll not need to provide full URLs, just the IP (for local) or prefix (for dynamic). There are more than just protocol differences between the two environments (as port and domain).